src/script_interface/constraints/Constraints.hpp

```diff
@@ -27,13 +27,15 @@
 #include "ScriptObjectRegistry.hpp"
 #include "core/constraints.hpp"
 #include "script_interface/ScriptInterface.hpp"
+#include "core/initialize.hpp"
 
 namespace ScriptInterface {
 namespace Constraints {
 
 class Constraints : public ScriptObjectRegistry<Constraint> {
   virtual void add_in_core(std::shared_ptr<Constraint> obj_ptr) override {
     ::Constraints::constraints.add(obj_ptr->constraint());
+    ::on_constraint_change();
```

Comment:
If this is needed you should probably also call it when a constraint is removed?

Reply:
True! Otherwise forces are not recalculated.